gossip/privdata/config.go

```diff
@@ -50,4 +65,24 @@ func (c *PrivdataConfig) loadPrivDataConfig() {
 
 	c.ReconciliationEnabled = viper.GetBool("peer.gossip.pvtData.reconciliationEnabled")
 
+	requiredPeerCount := viper.GetInt("peer.gossip.pvtData.implicitCollectionDisseminationPolicy.requiredPeerCount")
+
+	maxPeerCount := implicitCollectionMaxPeerCountDefault
+	if viper.Get("peer.gossip.pvtData.implicitCollectionDisseminationPolicy.maxPeerCount") != nil {
+		// allow override maxPeerCount to 0 that will effectively disable dissemination on the peer
+		maxPeerCount = viper.GetInt("peer.gossip.pvtData.implicitCollectionDisseminationPolicy.maxPeerCount")
+	}
```

Comment:
Why check for nil on maxPeerCount, but not on requiredPeerCount?

Reply:
Thank you, Dave. 
requiredPeerCount default value is 0 but maxPeerCount default value is 1. For requiredPeerCount, no matter if it is nil or 0, it will be 0. However, for maxPeerCount, we have to differentiate nil vs. 0, because maxPeerCount will be 1 (default value) if it is nil but will be 0 if it is 0. When maxPeerCount is set to 0 in core.yaml, it means to disable dissemination.